content/ops_guides/ha_guides/cloudify_ha_day_two_ops.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Cloudify cluster day 2 operations
+description: Guides for manging Cloudify HA clusters
+weight: 80
+alwaysopen: false
+---
+
+## Introduction
+
+Cloudify HA solution can contains three different clusters of database nodes, message queue brokers and Cloudify Managers.
```

Comment:
All day 2 operations should only be conducted while the cluster is in maintenance mode.